### PR TITLE
✨ Test - Add Playwright to Test Metadata 

### PR DESCRIPTION
**Pull request by Aibono1225**
Relates to #1178 

Added Playwright for UI testing, working as below:
- Check whether the https://ssw.com.au/rules/history.json file exists
- Go to latest rules page, randomly select a rule, navigate to that rule's page and check if it has metadata

image

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// require('dotenv').config();
+
+/**
+ * @see https://playwright.dev/docs/test-configuration
+ */
+module.exports = defineConfig({
+  testDir: './tests',
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
```
Comment (JackDevAU) [inline playwright.config.js:17]: is this something we need to add the env vars

File: .github/workflows/playwright.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Playwright Tests
+
+on:
+  pull_request:
```
Comment (JackDevAU) [inline .github/workflows/playwright.yml:4]: Should this also run with staging? so we can see this before Prod

File: playwright.config.js (hunk 1 above)
Comment (Aibono1225) [reply playwright.config.js:16]: Yes. Thanks for reminding me. I have added it in the newest commit

File: .github/workflows/playwright.yml (hunk 2 above)
Comment (Aibono1225) [reply .github/workflows/playwright.yml:4]: Done - it can be running on staging now.

**Review by JackDevAU — APPROVED** [review]
looks good 🚀

**Pull request merged**